homeassistant/components/motion_blinds/__init__.py

```diff
@@ -157,11 +158,14 @@ def stop_motion_multicast(event):
     else:
         version = f"Protocol: {motion_gateway.protocol}"
 
+    if TYPE_CHECKING:
+        assert entry.unique_id is not None
```

Comment:
It's a bit weird that we use the config entry unique_id for identifiers but for connections we use the gateway mac attribute. The config entry unique_id is set to the gateway mac address in the config flow.

Something for a future clean up perhaps.

Reply:
@starkillerOG is this something you could look into?

Reply:
@MartinHjelmare so you want to just use the gateway mac for the identifiers too, or do you want to remove the identifiers alltogether?
(I think the matching can be done with the connections also right? so the identifiers is not really needed I think...)

Reply:
Actually I tried removing the identifier here: https://github.com/starkillerOG/home-assistant/commit/58d464f3cb97c5ade49eb2c9dd368315c2663970
But the `via_device` does not work anymore when I test it.
It will not show anymore that the blinds are connected though the gateway in the device info page.

Reply:
The `via_device` item would need to be update too.

Reply:
Scratch that, the `via_device` matcher currently only matches via `indentifiers`. So I'd just simplify here and use gateway.mac in identifiers since that's the same as entry.unique_id.